proposals/initial-connection/README.md

```diff
@@ -202,341 +202,40 @@ hints for designing TDs. When to use this or not.
 
 TODO: Documenting why we have multi sec for one Thing. Some properties being public, some not. Reading being public, writing not.
 
-### Validation Rules
-
-- Reference for an object that doesn't exist: What happens in TD 1.1 is same here.
+## Validation Rules
+
+1. In an expanded TD, each form MUST contain the required information to construct the request, meaning an absolute `href`, `security`, `op` and protocol related information defined by the binding specification. If a term has a default value, it MUST be also provided.
```

Comment:
- @danielpeintner : Not sure if the expansion of the defaults is needed. More comments will be provided.
- @lu-zero the implementation will expand it anyway. This makes it explicit.

Reply:
```suggestion
1. In an expanded TD, each form MUST contain the information required to construct the request, meaning an absolute `href`, `security`, `op`, and protocol-related information defined by the binding specification. If a term has a default value, it MUST also be provided.
```